src/ceph-volume/ceph_volume/util/device.py

```diff
@@ -386,16 +386,20 @@ def used_by_ceph(self):
 
 
     def _check_generic_reject_reasons(self):
-        reasons = [
-            ('removable', 1, 'removable'),
-            ('ro', 1, 'read-only'),
-            ('locked', 1, 'locked'),
-        ]
-        rejected = [reason for (k, v, reason) in reasons if
-                    self.sys_api.get(k, '') == v]
-        # reject disks small than 5GB
-        if int(self.sys_api.get('size', 0)) < 5368709120:
-            rejected.append('Insufficient space (<5GB)')
+        rejected = []
+        if self.sys_api:
```

Comment:
Why is this condition needed? `sys_api` will at least be `{}` so we can always iterate over it.

Reply:
Indeed, fixed.